test.cpp

```diff
@@ -6,7 +6,39 @@ TEST(Evaluate, simple_addition_works) {
   EXPECT_EQ(2, evaluate("11+"));
 }
 
+void testSplitStringBySpace() {
+    // Test 1: Basic test
+    {
+        std::string input = "This is a test string";
+        std::vector<std::string> expected = {"This", "is", "a", "test", "string"};
+        assert(splitStringBySpace(input) == expected);
```

Comment:
is this what chatgpt emitted? :D

the test cases seem sane but please use the gtest framework like the other tests